Controller/HybridLogin.php

```diff
@@ -181,4 +194,33 @@ private function twitterLogin()
             $this->miniLog->error($exc->getMessage());
         }
     }
+
+    /**
+     * Manager FacturaScripts contact login
+     */
+    private function contactLogin()
+    {
+        if (AppSettings::get('webportal', 'allowlogincontacts', 'false') === 'false') {
+            return false;
+        }
+
+        $contactEmail = $this->request->request->get('fsContact', '');
+        if ($contactEmail !== '') {
+            $contact = new Contacto();
+            $where = [new DataBaseWhere('email', $contactEmail)];
+            if (!$contact->loadFromCode('', $where)) {
```

Comment:
if( not something ) { ... } else { ... } can be confused. Better to do:
if( something ) { ... } else { ... }